### PR TITLE
Add Autons

### PR DESCRIPTION
**Pull request by definatelyVincentWang**
Autons
* Fix PathWeaver directory
* Add ThreeBallAuton, FourBallAuton, FiveBallAuton folders into PathWeaver directory
* Add two paths for each: ThreeBallAuton, FourBallAuton, FiveBallAuton
* Create new OneBallAuton, TwoBallAuton, ThreeBallAuton, FourBallAuton, FiveBallAuton Auton Commands
* **Note that `withTimeout()` times are estimated placeholders and true times will be added after/during testing.**

Co-authored-by: Eric Lin <SovietNugget1@users.noreply.github.com>
Co-authored-by: William <thunder-red-star@users.noreply.github.com>
Co-authored-by: thegu5 <thegu5@users.noreply.github.com>
Co-authored-by: Ivan Wei <iwei20@users.noreply.github.com>
Co-authored-by: carminvuong <carminvuong@users.noreply.github.com>
Co-authored-by: marcjiang7 <marcjiang7@users.noreply.github.com>
Co-authored-by: ijiang05 <ijiang05@users.noreply.github.com>
Co-authored-by: Kelvin Zhao <t1rbr@users.noreply.github.com>
Co-authored-by: Ivan Chen <IvanChen07@gmail.com>
Co-authored-by: MdyWg <MdyWg@users.noreply.github.com>

### CODE REVIEW

File: src/main/java/com/stuypulse/robot/commands/auton/FiveBallAuton.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    // Time we want to give the robot.drivetrain to align
+    private static final int DRIVETRAIN_ALIGN_TIME = 2;
+
+    private static final int HUMAN_WAIT_TIME = 2;
+
+    // Timeouts for Pathweaver paths
+    private static final int FIVE_BALL_TO_SECOND_BALL_TIMEOUT = 10;
+    private static final int FIVE_BALL_TO_TERMINAL_TIMEOUT = 10;
+    private static final int FIVE_BALL_TERMINAL_TO_SHOOT_TIMEOUT = 10;
+    private static final int FIVE_BALL_TO_WALL_BALL_TIMEOUT = 10;
```
Comment (selym3) [inline src/main/java/com/stuypulse/robot/commands/auton/FiveBallAuton.java:48]: the ramsete commands are on a timer anyway, no need to add a with timeout. the alignment and shooting do need them

File: src/main/java/com/stuypulse/robot/commands/auton/TwoBallAuton.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+ * @author Ian Jiang (ijiang05@gmail.com)
+ * @author Carmin Vuong (carminvuong@gmail.com)
+ * @author Samuel Chen(samchen1738@gmail.com)
+ */
+
+public class TwoBallAuton extends SequentialCommandGroup {
+    // Time it takes for the shooter to reach the target speed
+    private static final int SHOOTER_INITIALIZE_DELAY = 1;
+    // Distance from starting position to ring shot
+    private static final int DISTANCE_TO_RING = 12;
```
Comment (selym3) [inline src/main/java/com/stuypulse/robot/commands/auton/TwoBallAuton.java:36]: what unit is this in?

File: src/main/java/com/stuypulse/robot/commands/auton/ThreeBallAuton.java [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    private static final int THREE_BALL_START_TIMEOUT = 10;
+    private static final int THREE_BALL_TO_TERMINAL_TIMEOUT = 10;
+    private static final int THREE_BALL_SHOOT_TERMINAL_BALLS_TIMEOUT = 10;
+
+    private static final String THREE_BALL_START =
+            "ThreeBallAuton/output/ThreeBallAutonGetSecondBall.wpilib.json";
+    private static final String THREE_BALL_TO_TERMINAL =
+            "ThreeBallAuton/output/ThreeBallAutonGetTerminalBalls.wpilib.json";
+    private static final String THREE_BALL_SHOOT_TERMINAL_BALLS = 
+            "ThreeBallAuton/output/FourBalAutonShootTerminalBalls.wpilib.json";
```
Comment (selym3) [inline src/main/java/com/stuypulse/robot/commands/auton/ThreeBallAuton.java:50]: i imagine that this is not the name of the three ball path

File: src/main/java/com/stuypulse/robot/commands/auton/FourBallAuton.java [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    private static final int FOUR_BALL_START_TIMEOUT = 10;
+    private static final int FOUR_BALL_TO_TERMINAL_TIMEOUT = 10;
+    private static final int FOUR_BALL_SHOOT_TERMINAL_BALLS_TIMEOUT = 10;
+
+    private static final String FOUR_BALL_START =
+            "FourBallAuton/output/FourBallAutonGetSecondBall.wpilib.json";
+    private static final String FOUR_BALL_TO_TERMINAL =
+            "FourBallAuton/output/FourBallAutonGetTerminalBalls.wpilib.json";
+    private static final String FOUR_BALL_SHOOT_TERMINAL_BALLS = 
+            "FourBallAuton/output/FourBalAutonShootTerminalBalls.wpilib.json";
```
Comment (selym3) [inline src/main/java/com/stuypulse/robot/commands/auton/FourBallAuton.java:52]: four ball

**Comment by selym3** [comment]
another comment, can u figure out why the PathWeaver subdirectories are inconsistent (e..g pathweaver folder in four and five bals but not threeball

**Comment by definatelyVincentWang** [comment]
I think I might've deleted the PathWeaver file. Maybe PathWeaver doesn't need the Auto and Path folders to be in a directory explicitly named PathWeaver, which is why it still worked and I could still open it in PathWeaver.

**Review by selym3 — CHANGES_REQUESTED** [review]

**Review by Sam-Belliveau — CHANGES_REQUESTED** [review]

**Pull request merged**